### PR TITLE
Update README with relevant information

### PR DESCRIPTION
**Pull request by dbampalikis**
Added some basic documentation about the theme and the way to publish changes and new material

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,89 +1,12 @@
...
 
-[![license](https://img.shields.io/github/license/kitian616/jekyll-TeXt-theme.svg)](https://github.com/kitian616/jekyll-TeXt-theme/blob/master/LICENSE)
-[![Gem Version](https://img.shields.io/gem/v/jekyll-text-theme.svg)](https://github.com/kitian616/jekyll-TeXt-theme/releases)
-[![Travis](https://img.shields.io/travis/kitian616/jekyll-TeXt-theme.svg)](https://travis-ci.org/kitian616/jekyll-TeXt-theme)
-[![Tip Me via PayPal](https://img.shields.io/badge/PayPal-tip%20me-1462ab.svg?logo=paypal)](https://www.paypal.me/kitian616)
-[![Tip Me via Bitcoin](https://img.shields.io/badge/Bitcoin-tip%20me-f7931a.svg?logo=bitcoin)](https://raw.githubusercontent.com/kitian616/jekyll-TeXt-theme/master/docs/assets/images/3Fkufxcw2xd8HnaRJBNK4ccdtkUDyyNu4V.jpg)
+The repository contains the codebase for the Swedish Sensitive Data Archive web portal. It provides information about the submission and retrieval of a dataset stored in the Swedish SDA.
 
-![TeXt Theme](https://raw.githubusercontent.com/kitian616/jekyll-TeXt-theme/master/screenshots/TeXt-home.jpg)
+## Jekill Theme
```
Comment (norling) [inline README.md]: typo:
```suggestion
## Jekyll Theme
```

File: README.md [hunk 2]
```diff
@@ -1,89 +1,12 @@
...
 
-![TeXt Theme Details](https://raw.githubusercontent.com/kitian616/jekyll-TeXt-theme/master/screenshots/TeXt-layouts.png)
+The portal is using the [TeXt-theme](https://github.com/kitian616/jekyll-TeXt-theme) template. Documentation about the template and different types of pages available can be found [here](https://tianqi.name/jekyll-TeXt-theme/docs/en/quick-start). Currently, all the page are created with `layout: article`. The navigation bar is configured in the file `_data/navigation.yml`.
```
Comment (norling) [inline README.md]: Not important, but I think this reads easier like
```suggestion
The portal is using the [TeXt-theme](https://github.com/kitian616/jekyll-TeXt-theme) template. Documentation about the template and the different types of available pages can be found [here](https://tianqi.name/jekyll-TeXt-theme/docs/en/quick-start). Currently, all the page are created with `layout: article`. The navigation bar is configured in the file `_data/navigation.yml`.
```

File: README.md [hunk 3]
```diff
@@ -1,89 +1,12 @@
...
 
-TeXt is a super customizable Jekyll theme for personal site, team site, blog, project, documentation, etc. Similar to iOS 11 style, it has large and prominent titles, round buttons and cards.
+## Publishing changes
 
-**[Change Log](https://github.com/kitian616/jekyll-TeXt-theme/blob/master/CHANGELOG.md)** | **[中文](https://github.com/kitian616/jekyll-TeXt-theme/blob/master/README-zh.md)**
+According to the current setup, the code in the master branch is published. To change the web portal, edit and/or add a new `.md` file in the root folder and push the changes. Merging the changes to the master branch enables the action that updates the site.
```
Comment (norling) [inline README.md]: These sentences were a bit hard to read for me. 
```suggestion
Currently the page is published from the `master` branch. To make changes, edit and/or add a new `.md` file in the root folder and push the changes. Once the changes are merged to the master branch, the page will update. Note that the actual update may take a few minutes as it needs to wait for the github to render the new changes.

**Review by norling — APPROVED** [review]
Looks good! :) I had some nitpicky suggestions you can take a look at, but nothing major!

**Review by viklund — APPROVED** [review]

**Pull request merged**